pkg/ui/key_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/xiaq/persistent/hash"
 )
 
-var kTests = []struct {
+var identityTests = []struct {
```

Comment:
This is the list of test cases for the function `K`, so `kTests` is more accurate.